src/Symfony/Component/RateLimiter/Tests/RateLimiterTest.php

```diff
@@ -27,7 +27,7 @@ class RateLimiterTest extends TestCase
      */
     public function testValidConfig(string $expectedClass, array $config)
     {
-        $factory = new RateLimiter($config, new InMemoryStorage());
+        $factory = new RateLimiterFactory($config, new InMemoryStorage());
```

Comment:
This `RateLimiterTest` class and file should be renamed

Reply:
Thank you. 
These were added after I made the PR. I forgot these.